### ISSUE TITLE
Drawer - Bottom variation doesn't overlay

### ISSUE THREAD

**Issue by kahboom** [patternfly/patternfly-react#7678]
Hello, I've been using the Drawer component "right" and "left" variations, which act like an overlay on top of the `DrawerContent`. I'm trying to use the "bottom" variation in the same way, to show a collapsible Log Viewer. The problem is that, unlike the other positions, the bottom Drawer seems to create a space for it underneath the `DrawerContent`, which breaks the page height (right now it takes the full inner height of the window, so there is no scrolling, but with the bottom Drawer that's no longer the case).

Is there any way to override this behavior so it doesn't require that extra spacing on the bottom somehow?

In other words, I'm looking for a way to grow a sticky footer without increasing the viewport. Something like this: https://codepen.io/kahboom/pen/GRxqoQL

**Comment by mcoker** [comment]
Hey @kahboom! We don't have a demo of this in react, but we have a [bottom drawer example](https://www.patternfly.org/v4/components/drawer/html-demos#expanded-bottom) in the core docs that shows the structure of how this would work.

I created a working codesandbox here (the bell icon in the header toggles the drawer): https://codesandbox.io/s/dreamy-nash-8hhuoc?file=/index.html

Let me know if that works. I can also put together a react demo if you'd like.

**Comment by kahboom** [comment]
Hey @mcoker ! Thanks, that helps a lot.

**Comment by mcoker** [comment]
@kahboom you're welcome! In case you run into this again, a gotcha with the bottom variation is that the drawer parent needs some sort of height set for the drawer/panel height to work properly. We've set it up so that it works well with the page component.

However I'm thinking we can get rid of that requirement so the bottom variation is more flexible. Created an issue to investigate that further - https://github.com/patternfly/patternfly/issues/4975
